### PR TITLE
Fix guide for Ballerina 0.990

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
- Fix for latest syntax changes
- Refactor code

### CODE REVIEW

File: notification-sender/notification_sender.bal [hunk 1]
```diff
@@ -83,45 +82,53 @@ public function main(string... args) {
 }
 
 # Send notification to the customers.
+#
 # + return - State of whether the process of sending notification is success or not
```
Comment (arukshani) [inline notification-sender/notification_sender.bal]: Shall we rephrase this to say `A boolean indicating the status of the send notification process`

Comment (ldclakmal) [reply notification-sender/notification_sender.bal]: Fixed with https://github.com/ballerina-guides/gmail-spreadsheet-integration/pull/20/commits/231df90d595dfb0824c5eaee49acb73cfb6451e6

File: README.md [hunk 2]
```diff
@@ -200,25 +191,30 @@ The main function in `notification_sender.bal` calls `sendNotification` function
...
-            if (!isSuccess) {
-                break;
+    var customerDetails = getCustomerDetailsFromGSheet();
+    if (customerDetails is error) {
+        log:printError("Failed to retrieve customer details from GSheet", err = customerDetails);
+        return false;
+    } else {
+        int i = 0;
+        boolean isSuccess = false;
+        //Iterate through each customer details and send customized email.
```
Comment (samgnaniah) [inline README.md]:
```suggestion
        //Iterate through each customer's details and sends customized email.
```

File: README.md [hunk 3]
```diff
@@ -248,17 +244,17 @@ The following is a sample email body.
...
-INFO  [wso2.notification-sender] - Gmail-Google Sheets Integration -> Email sending process successfully completed! 
+INFO  [wso2.notification-sender] - GMail-Google Sheets Integration -> Email sending process successfully completed!
 
 ```
 
 ### Writing unit tests    
 
-In Ballerina, the unit test cases should be in the same package inside a folder named as 'tests'.  When writing the test 
+In Ballerina, the unit test cases should be in the same module inside a folder named as 'tests'.  When writing the test
 functions the below convention should be followed.
```
Comment (samgnaniah) [inline README.md]:
```suggestion
functions, the following convention should be followed.
```
